pkgs/os-specific/linux/openvswitch/update.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i nu -p common-updater-scripts
```

Comment:
```suggestion
#!nix-shell -i nu -p nushell common-updater-scripts
```

Not everyone has nushell already installed!

Reply:
Oops, I had nushell in originally as the interpreter, but that's a command not a package and I never added it to the package list. Fixed.